options.cpp

```diff
@@ -278,30 +278,14 @@ std::string database_options_t::conninfo() const
 }
 
 options_t::options_t()
-: prefix("planet_osm"),
-  projection(reprojection::create_projection(PROJ_SPHERE_MERC)), append(false),
-  slim(false), cache(800), tblsmain_index(boost::none),
-  tblsslim_index(boost::none), tblsmain_data(boost::none),
-  tblsslim_data(boost::none), style(DEFAULT_STYLE),
-  expire_tiles_zoom(0), expire_tiles_zoom_min(0),
-  expire_tiles_max_bbox(20000.0), expire_tiles_filename("dirty_tiles"),
-  hstore_mode(HSTORE_NONE), enable_hstore_index(false), enable_multi(false),
-  hstore_columns(), keep_coastlines(false), parallel_indexing(true),
+: projection(reprojection::create_projection(PROJ_SPHERE_MERC)),
 #ifdef __amd64__
   alloc_chunkwise(ALLOC_SPARSE | ALLOC_DENSE),
 #else
   alloc_chunkwise(ALLOC_SPARSE),
 #endif
```

Comment:
Want to get rid of the #ifdef in the middle of the constructor as well?

Reply:
Nope. There is another place in the code with the same 'ifdef' and I haven't fully understood what's going on there, so I'd rather leave this for now.